_build/html/_sources/capstone-report.md

```diff
@@ -104,12 +107,9 @@ name: true_pos_all_wrong
 ---
 A manual inspection of tricky examples was conducted. Above, we have a true positive and although, all model are struggling, DenseNet is the least confident in its wrong prediction.  
 ```
+In an effort to make the model more generalizable, we considered various techniques such as implemeting dropout layers in our CNN structure and taking an ensemble approach. Further, the medical nature of this project suggests that reducing false negatives (optimizing recall) may be more important than maximizing the overall accuracy on postive and negative predictions. To find more about these explorations, see [Appendix B - Documentation](../documentation.md).
```

Comment:
Spelling: "implementing"

Suggestion:
"In an effort to make the model more generalizable, we considered various techniques such as implementing dropout layers in our CNN structure, batch normalization and taking an ensemble approach. Furthermore, when identifying liphypertrophic masses, it is more detrimental for a true positive (areas where lipohypertrophy is present) to be falsely labelled as a negative. This is formally known as a false negative and thus, we also attempted to optimize recall, a score where a higher value indicates fewer false negatives. To find out more about these explorations and their outcomes, see [Appendix B - Documentation](../documentation.md)."

Reply:
Great suggestion, thank you!